bottles/frontend/ui/list-entry.blp

```diff
@@ -84,7 +84,6 @@ template BottleViewEntry : .AdwActionRow {
   }
 
   Button btn_run {
-    tooltip-text: _("Run .exe/.msi in this bottle");
```

Comment:
What's the reason behind this?

Reply:
It's been moved to the Python code, so the name of the bottle is in the tooltip